schemas/stsci.edu/asdf/0.1.0/time/time.yaml

```diff
@@ -0,0 +1,232 @@
+              section is optional.
+
+            - `yday`: Year, day-of-year and time as
+              `YYYY:DOY:HH:MM:SS.sss...`. The day-of-year (DOY) goes
+              from 001 to 365 (366 in leap years). For example,
+              `2000:001:00:00:00.000` is midnight on January 1,
+              2000.
+
+            - `byear`: Besselian Epoch year, eg. `B1950.0`.  The `B`
+              is optional if the `byear` format is explicitly
+              specified.
+
+            - `jyear`: Julian Epoch year, eg. `J2000.0`.  The `J` is
+              optional if the `jyear` format is explicitly
+              specified.
+
+            - `decimalyear`: Time as a decimal year, with integer
+              values corresponding to midnight of the first day of
+              each year. For example 2000.5 corresponds to the ISO
+              time `2000-07-02 00:00:00`.
+
+            - `jd`: Julian Date time format. This represents the
+              number of days since the beginning of the Julian
+              Period. For example, 2451544.5 in `jd` is midnight on
+              January 1, 2000.
+
+            - `mjd`: Modified Julian Date time format. This
+              represents the number of days since midnight on
+              November 17, 1858. For example, 51544.0 in MJD is
+              midnight on January 1, 2000.
+
+            - `gps`: GPS time: seconds from 1980-01-06 00:00:00 UTC
+              For example, 630720013.0 is midnight on January 1,
+              2000.
+
+            - `unix`: Unix time: seconds from 1970-01-01 00:00:00
+              UTC. For example, 946684800.0 in Unix time is midnight
+              on January 1, 2000.  [TODO: Astropy's definition of
+              UNIX time doesn't match POSIX's here.  What should we
+              do for the purposes of ASDF?]
+
+            - `cxcsec`: Chandra X-ray Center seconds from 1998-01-01
+              00:00:00 TT. For example, 63072064.184 is midnight on
+              January 1, 2000.  [TODO: Is this general enough to
+              include?]
+
+          enum:
+            - iso
+            - yday
+            - byear
+            - jyear
+            - decimalyear
+            - jd
+            - mjd
+            - gps
+            - unix
+            - cxcsec
+
+        scale:
+          description: |
+            The time scale (or time standard) is a specification for
+            measuring time: either the rate at which time passes; or
+            points in time; or both. See also [3] and [4].
+
+            [TODO: Describing these scales in enough detail to
+            reimplement is perhaps too much to do here, but we should
+            have good ironclad references for each.]
+
+            The supported time scales are:
+
+            - `utc`: Coordinated Universal Time (UTC).  This is the
+              default time scale, except for the formats that
+              represents seconds since an epoch, namely `gps`, `unix`
```

Comment:
The epoch_scale for `unix` is UTC, so that one (by chance) also defaults to UTC.  So maybe just say `This is the default time scale except for 'gps' and 'unix'`.